types/process.go

```diff
@@ -44,6 +44,10 @@ type FileDescriptor interface {
 	FileDescriptorCount() (int, error)
 }
 
+type OpenHandles interface {
```

Comment:
Please add godocs with all new types and methods. The project has some tech debt in this area but no need to add to it.